httpx_oauth/clients/twitch.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, List, Optional, Tuple, cast
+
+from httpx_oauth.errors import GetIdEmailError
+from httpx_oauth.oauth2 import BaseOAuth2
+
+AUTHORIZE_ENDPOINT = "https://id.twitch.tv/oauth2/authorize"
+ACCESS_TOKEN_ENDPOINT = "https://id.twitch.tv/oauth2/token"
+REVOKE_TOKEN_ENDPOINT = "https://id.twitch.tv/oauth2/revoke"
+BASE_SCOPES = [
+    "user:read:email",
+    "user:read:follows",
+    "user:read:subscriptions",
+    "user:manage:whispers",
+]
+
+"""
+If an ID or login name is not specified, the request to the user endpoint returns 
+information about the user specified in the included access token.
+
+https://dev.twitch.tv/docs/api/reference#get-users
+"""
+PROFILE_ENDPOINT = "https://api.twitch.tv/helix/users"
+
+LOGO_SVG = """
```

Comment:
For consistency with other logos, I prefer we use this version: https://github.com/gilbarbara/logos/blob/master/logos/twitch.svg?short_path=7cb962f

Reply:
No problem, fixed -> https://github.com/frankie567/httpx-oauth/pull/298/commits/dd071a1039206eac05800f52f682ba07921c250d